src/main/java/edu/umich/eecs/april/apriltag/ApriltagDetectorActivity.java

```diff
@@ -128,11 +128,12 @@ protected void onResume() {
         double decimation = Double.parseDouble(sharedPreferences.getString("decimation_list", "8"));
         double sigma = Double.parseDouble(sharedPreferences.getString("sigma_value", "0"));
         int nthreads = Integer.parseInt(sharedPreferences.getString("nthreads_value", "4"));
+        int max_hamming_error = Integer.parseInt(sharedPreferences.getString("max_hamming_error", "0"));
```

Comment:
Good catch, that must've been causing the FPs. Although IIRC the default at April was something > 0 for robustness to noise(?)

Reply:
the default hardcoded value when we init the apriltag detector was 2 here. probably was based on 36h11, where 2 error bits is still totally likely to be a tag -- but for 16h5, that's just _so_ noisy.

Reply:
Oh yes, it's obvious now that I try it. So many false detections from noise